nautobot/docs/models/extras/relationship.md

```diff
@@ -22,14 +22,20 @@ To create a relationship, from the top-level navigation menu select **Extensibil
 !!! note
     A symmetric many-to-many relationship can be, but is not necessarily, a _complete graph_ or _full mesh_. For example, in the routing topology example above, if Device _A_ and Device _B_ are peers, and Device _B_ and Device _C_ are peers, this does not automatically imply a relationship between Devices _A_ and _C_ -- they **might or might not** also be peers, depending on how you define and populate the specific associations for this relationship.
 
-## Requiring a Relationship
```

Comment:
Is there a reason we deleting this heading?

I feel we could leave the version admonition in but just as a line.